.github/workflows/publish-runtime.yml

```diff
@@ -0,0 +1,166 @@
+name: Publish Runtime Draft
+
+# The code (like generate-release-body) will be taken from the tag versions, not master
+on:
+  workflow_dispatch:
+    inputs:
+      from:
+        description: tag (ex. runtime-53) to retrieve commit diff from
+        required: true
+      to:
+        description: tag (ex. runtime-155) to generate release note and srtool runtimes from
+        required: true
+
+jobs:
+  ####### Build runtimes with srtool #######
+
+  setup-scripts:
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
If you want more comments about this, it is to ensure the script are executed from the current branch and not from the tags being checked out later in the process